test/bdd/pkg/verifiable/verifiable_steps.go

```diff
@@ -95,44 +98,65 @@ func (s *SDKSteps) createVC(issuedAt, subject, issuer string) (*verifiable.Crede
 	return vcToIssue, nil
 }
 
-func (s *SDKSteps) addVCProof(vc *verifiable.Credential, issuer, proofType string) error {
+func (s *SDKSteps) addVCProof(vc *verifiable.Credential, issuer, proofType string) ([]byte, error) {
 	doc := s.getPublicDID(issuer)
 	pubKey := doc.PublicKey[0]
 
 	kms := s.bddContext.AgentCtx[issuer].Signer()
 
+	s.proofType = proofType
+
 	switch proofType {
 	case jwsLinkedDataProof:
 		creator := doc.ID + pubKey.ID
 
 		err := addJWSLinkedDataProof(vc, newSigner(kms, base58.Encode(pubKey.Value)), creator)
 		if err != nil {
-			return err
+			return nil, err
+		}
+
+		return vc.MarshalJSON()
+
+	case jwsProof:
+		// TODO to be improved at https://github.com/hyperledger/aries-framework-go/issues/339:
+		//  1) use KMS for signing
```

Comment:
KMS and Crypto are now available in the code.. they should be referenced here as opposed to GenerateKey()

also, use NIST approved curves like P-256 instead of ed25519 by default

Reply:
Yeah, I see. I will use crypto as I did for JWS linked data proof here in the https://github.com/hyperledger/aries-framework-go/pull/1354